### ISSUE TITLE
Android build fails with new React Native 0.55.3 project

### ISSUE THREAD

**Issue by lukebaker** [pushbots/pushbots-react-native#2]
I created a new React Native 0.55.3 project, added the pushbots-react-native module, made the Android changes listed in the README, and ran `react-native run-android`. I received the following error:

```
/tmp/blah/node_modules/pushbots-react-native/android/src/main/java/com/reactlibrary/RNPushbotsModule.java:53: error: cannot find symbol
       if(value != null && !value.isEmpty()){
          ^
  symbol:   variable value
  location: class RNPushbotsModule
/tmp/blah/node_modules/pushbots-react-native/android/src/main/java/com/reactlibrary/RNPushbotsModule.java:53: error: cannot find symbol
       if(value != null && !value.isEmpty()){
                            ^
  symbol:   variable value
  location: class RNPushbotsModule
2 errors
:pushbots-react-native:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.
```

FWIW, I also received this same error when attempting to integrate the pushbots library into an existing React Native 0.55.2 project.

Here's the method in question:
https://github.com/pushbots/pushbots-react-native/blob/74fa34323c6ec86087831c250bafb94910d3724a/android/src/main/java/com/reactlibrary/RNPushbotsModule.java#L51-L56

And here's where those lines were added: https://github.com/pushbots/pushbots-react-native/commit/74fa34323c6ec86087831c250bafb94910d3724a

**Comment by amrsobhy** [comment]
thanks for reporting this. I have assigned this to one of our technical team and you can expect both a response and a fix before end of day. ​

**Comment by AbdullahDiaa** [comment]
Fixed in 1.0.6
